tfhe/src/shortint/engine/server_side/comp_op.rs

```diff
@@ -90,23 +79,13 @@ impl ShortintEngine {
         ct_left: &mut Ciphertext,
         ct_right: &mut Ciphertext,
     ) -> EngineResult<Ciphertext> {
-        let mut result = ct_left.clone();
-        self.smart_greater_or_equal_assign(server_key, &mut result, ct_right)?;
-        Ok(result)
-    }
-
-    pub(crate) fn smart_greater_or_equal_assign(
-        &mut self,
-        server_key: &ServerKey,
-        ct_left: &mut Ciphertext,
-        ct_right: &mut Ciphertext,
-    ) -> EngineResult<()> {
         if !server_key.is_functional_bivariate_pbs_possible(ct_left, ct_right) {
```

Comment:
same thing throughout looks like the allow is not required anymore

Reply:
Removed